conda/shell/condabin/_conda_activate.bat

```diff
@@ -18,6 +18,7 @@
   FOR %%A IN ("%~dp0.") DO @SET _sysp=%%~dpA
   IF NOT EXIST "!_sysp!\Scripts\conda.exe" @SET "_sysp=!_sysp!..\"
 )
+@FOR %%A in ("%TMP%") do @SET TMP=%%~sA
```

Comment:
Are we sure that this approach will work?  What happens if the original TMP is something like "C:\Users\My User\AppData\tmp" and this shortens it to "C:\Users\MyUser\AppData\tmp" - surely that won't work, right?

Reply:
The ~s "Expands path to contain short names only." https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/for

So in my testing with that set, "C:\Users\Test User\AppData\Local\Temp" becomes "C:\Users\TESTUS~1\AppData\Local\Temp"

Reply:
Sounds good.  Thanks!